.github/workflows/sqa-tools-builder.yaml

```
@@ -0,0 +1,115 @@
+name: Build SQA Tools
+
+on:
+  push:
+# TODO add these lines back       branches:
+#            - main
+#            - "release_*"
+  pull_request:
+      branches:
+          - main
+          - "release_*"
+
+concurrency:
+    group: sqa-tools-builder-${{ github.ref }}
+    cancel-in-progress: true
+
+env:
+  PW_ENVIRONMENT_ROOT: ${{ github.workspace }}/.environment
+
+jobs:    
+  build-sqa-tools:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/project-chip/chip-build-crosscompile:latest
+    steps:
```

just to add a bit of spacing between configurations and steps
```suggestion
    
    steps:
```